src/core/logical_layers/renderers/stylesConfigs/mcda/mcdaStyle.ts

```
@@ -58,9 +58,9 @@ export function linearNormalization(
   layers: MCDAConfig['layers'],
 ): ExpressionSpecification {
   if (layers.length === 1) {
-    return ['/', calculateLayer(layers.at(0)!), layers.at(0)!.coefficient];
+    return ['/', calculateMCDALayer(layers.at(0)!), layers.at(0)!.coefficient];
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)